merkle/compact/range_test.go

```diff
@@ -213,7 +214,8 @@ func TestGoldenRanges(t *testing.T) {
 			if want := roots[size]; !bytes.Equal(hash, want) {
 				t.Errorf("root hash mismatch: got %x, want %x", hash, want)
 			}
-			if diff := pretty.Compare(cr.Hashes(), hashes[size]); diff != "" {
+			// TODO(): Remove equate empty
+			if diff := cmp.Diff(cr.Hashes(), hashes[size], cmpopts.EquateEmpty()); diff != "" {
```

Comment:
@pavelkalinnikov - this line worries me. Could you double check the accuracy of compact range?

Reply:
The compact range code is fine: a default constructed `compact.Range` has a default constructed `hashes` slice, i.e. `null`. So, if you set [this](https://github.com/google/trillian/blob/737aff5a2e691fdd515617d362887afb2e63effe/merkle/testonly/constants.go#L86) line to `nil`, `EquateEmpty` shouldn't be needed. Let me know if that breaks anything else.